### DISCUSSION TITLE
Welcome to PILOT Drive Discussions!

### DISCUSSION THREAD

**Discussion by lamemakes** [lamemakes/pilot-drive#12] [Announcements]
Ask questions, propose some new features, and discuss bugs/improvements!

**Comment by lymanc** [comment]
At this pointed I have installed dash-drive on Ubuntu 22.04 and rpi 4, and in both cases was able to access it on my browser. Howeve in both cases after changing something in "setup", both instances are no longer operative. Any thoughts on cause and how to fix?  

Some back ground. My interest is to have Android Auto like capabilities in my classic 1971 BMW Bavaria without molesting the dash or center console.  To this end I have found a way to mount a thin tablet such as an Amazon Fire, or Google Pixel Slate, and if addtional electronics are needed such a computer, router, or whatever they would be mounted under the rear seat. My interest in dash-drive is more of a user than developer as my development skills a quite dated.
[pilot-note.txt](https://github.com/lamemakes/pilot-drive/files/12065068/pilot-note.txt)

**Reply by lamemakes** [reply]
Hey @lymanc, this sounds like a really cool project!

> Howeve in both cases after changing something in "setup", both instances are no longer operative.

Can you explain your setup process and what specific changes you made? Not sure I completely understand what you mean here. You also reference "dash-drive" a few times, this is PILOT Drive - just want to confirm you have the right project here before we start troubleshooting.

Lastly could start a new discussion copy & pasting your comment [here](https://github.com/lamemakes/pilot-drive/discussions/categories/troubleshooting)? Just for organization and visibility to other users that might have this same issue.

Thanks for giving PILOT Drive a go!

**Reply by lamemakes** [reply]
Locking this announcement to further comments, all troubleshooting issues should create a new discussion in the [Troubleshooting](https://github.com/lamemakes/pilot-drive/discussions/categories/troubleshooting) section.
